crates/ruff_python_parser/src/python.lalrpop

```diff
@@ -669,8 +669,8 @@ LiteralPattern: ast::Pattern = {
         value: Box::new(value.into()),
         range: (location..end_location).into()
     }.into(),
-    <location:@L> <strings:StringLiteralOrFString+> <end_location:@R> =>? Ok(ast::PatternMatchValue {
-        value: Box::new(concatenate_strings(strings, (location..end_location).into())?.into()),
+    <location:@L> <strings:StringLiteralOrFString+> <end_location:@L> =>? Ok(ast::PatternMatchValue {
```

Comment:
Is using `@L` for the `end_location` intentional?

Reply:
Oh, it must be a typo. Thanks for pointing it out. This reminds me that I should write some test cases around strings used in match cases and mapping keys.